lib/src/components/InputText/input-text.module.scss

```diff
@@ -1,7 +1,6 @@
 @layer components {
   .input-text-wrapper {
     position: relative;
-    width: 100%;
```

Comment:
Maybe we could expose and override a custom prop (`--inputWrapperWidth`) for the slider because this prevents inputs from having a width of 100%
<img width="971" height="606" alt="Screenshot 2025-09-30 at 10 39 49" src="https://github.com/user-attachments/assets/59791e91-9630-49a7-95f7-6ab8b7f2ccb1" />

Reply:
Thanks 🙏 [3c1e247](https://github.com/WTTJ/welcome-ui/pull/2922/commits/3c1e247b73cc3187169c72178f81b4fa17220358)